fuel-vm/src/interpreter/debug.rs

```diff
@@ -63,140 +57,192 @@ where
     }
 }
 
-#[test]
-fn breakpoint_script() {
-    use fuel_asm::op;
-    use fuel_tx::ConsensusParameters;
-
-    let mut vm = Interpreter::<_, _, _>::with_memory_storage();
-
-    let gas_limit = 1_000_000;
-    let gas_price = 0;
-    let height = Default::default();
-
-    let script = [
-        op::addi(0x10, RegId::ZERO, 8),
-        op::addi(0x11, RegId::ZERO, 16),
-        op::addi(0x12, RegId::ZERO, 32),
-        op::addi(0x13, RegId::ZERO, 64),
-        op::addi(0x14, RegId::ZERO, 128),
-        op::ret(0x10),
-    ]
-    .into_iter()
-    .collect();
-
-    let consensus_params = ConsensusParameters::standard();
-
-    let tx = TransactionBuilder::script(script, vec![])
-        .script_gas_limit(gas_limit)
-        .add_random_fee_input()
-        .finalize()
-        .into_checked(height, &consensus_params)
-        .expect("failed to generate checked tx")
-        .into_ready(
-            gas_price,
-            consensus_params.gas_costs(),
-            consensus_params.fee_params(),
-        )
-        .unwrap();
-
-    let suite = vec![
-        (
-            Breakpoint::script(0),
-            vec![(0x10, 0), (0x11, 0), (0x12, 0), (0x13, 0), (0x14, 0)],
-        ),
-        (
-            Breakpoint::script(2),
-            vec![(0x10, 8), (0x11, 16), (0x12, 0), (0x13, 0), (0x14, 0)],
-        ),
-        (
-            Breakpoint::script(3),
-            vec![(0x10, 8), (0x11, 16), (0x12, 32), (0x13, 0), (0x14, 0)],
-        ),
-        (
-            Breakpoint::script(5),
-            vec![(0x10, 8), (0x11, 16), (0x12, 32), (0x13, 64), (0x14, 128)],
-        ),
-    ];
-
-    suite.iter().for_each(|(b, _)| vm.set_breakpoint(*b));
-
-    let state = vm
-        .transact(tx)
-        .map(ProgramState::from)
-        .expect("Failed to execute script!");
-
-    suite
+#[cfg(test)]
+mod tests {
+    use alloc::{
+        vec,
+        vec::Vec,
+    };
+
+    use super::Interpreter;
+    use crate::prelude::*;
+    use fuel_asm::RegId;
+
+    #[test]
+    fn breakpoint_script() {
+        use fuel_asm::op;
+        use fuel_tx::ConsensusParameters;
+
+        let mut vm = Interpreter::<_, _, _>::with_memory_storage();
+
+        let gas_limit = 1_000_000;
+        let gas_price = 0;
+        let height = Default::default();
+
+        let script = [
+            op::addi(0x10, RegId::ZERO, 8),
+            op::addi(0x11, RegId::ZERO, 16),
+            op::addi(0x12, RegId::ZERO, 32),
+            op::addi(0x13, RegId::ZERO, 64),
+            op::addi(0x14, RegId::ZERO, 128),
```

Comment:
It would be nice to use some constant names for registers. It will connect them here with the values in `suite`